### ISSUE TITLE
Ugrids without time fail

### ISSUE THREAD

**Issue by rsignell-usgs** [sci-wms/sci-wms#132]
This UGRID dataset contains only depth, so there is no time coordinate:
https://gamone.whoi.edu/thredds/dodsC/usgs/vault0/models/tides/ec2015/depth.nc.html
and when we try to add this to sci-wms, `getMap` requests fail because they are looking for time:
```python-traceback
...
  File "/srv/sci-wms/wms/models/datasets/ugrid.py", line 208, in getmap
    time_index, time_value = self.nearest_time(layer, request.GET['time'])
  File "/srv/sci-wms/wms/models/datasets/netcdf.py", line 235, in nearest_time
    time_var_name = find_appropriate_time(var_obj, time_vars)
  File "/srv/sci-wms/wms/utils.py", line 111, in find_appropriate_time
    raise ValueError('Unable to determine an appropriate variable to use as time.')
ValueError: Unable to determine an appropriate variable to use as time.
```

**Comment by rsignell-usgs** [comment]
So the workaround is to add a bogus time variable to get sci-wms to work.  But then you get time in the `getCapabilities`, and time controls in clients that should not be there:
![2018-04-05_8-06-19](https://user-images.githubusercontent.com/1872600/38364943-90aa0dd8-38a8-11e8-9032-54805d81efef.jpg)

**Comment by rsignell-usgs** [comment]
Just a check, I tried the depth dataset dap endpoint 
https://gamone.whoi.edu/thredds/dodsC/usgs/vault0/models/tides/ec2015/depth.nc
into ncWMS2 to check it would render as UGRID.  It does (although this plot took several minutes!):
![2018-04-05_8-25-30](https://user-images.githubusercontent.com/1872600/38365647-1cd50360-38ab-11e8-8e44-1b94422e0b69.jpg)

**Comment by kwilcox** [comment]
Fixed in 2.0
